src/main/java/core/basesyntax/app/FruitShopApp.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.app;
+
+import core.basesyntax.service.file.FileReader;
+import core.basesyntax.service.file.FileWriter;
+import core.basesyntax.service.parser.DataParser;
+import core.basesyntax.service.performer.Performer;
+import core.basesyntax.service.reporter.Reporter;
+
+public class FruitShopApp {
+    private final String dataFile;
+    private final String resultFile;
+    private final FileWriter fileWriter;
+    private final FileReader fileReader;
+    private final Reporter reporter;
+    private final Performer performer;
+    private final DataParser dataParser;
+
+    public FruitShopApp(String dataFile, String resultFile,
+                        FileWriter fileWriter, FileReader fileReader,
+                        Reporter reporter, Performer performer,
+                        DataParser dataParser) {
+        this.dataFile = dataFile;
+        this.resultFile = resultFile;
```

Comment:
better to pass this arguments to `createDailyReport`, because now your `FruitShopApp` work only with 2 files, which were setted while creation of `FruitShopApp`